recipe/meta.yaml

```diff
@@ -43,8 +43,8 @@ requirements:
     - python
     - numpy
     - scotch >=7
-    - mfront 4.1.0
-    - mgis 2.1
+    - mfront 4.2
+    - mgis 2.2
```

Comment:
Not sure why these were pinned, or if it's 100% necessary. In any case, with the old versions we cannot (easily) rebuild for newer boost, because mfront/mgis have only been rebuilt for 1.84 for their newest versions (unless we open a maintenance branch for 4.1 / 2.1 and backport the boost migrator).